### PR TITLE
fuzz: fix XML in website PUT.

### PR DESCRIPTION
**Pull request by robbat2**

### CODE REVIEW

File: request_decision_graph.yml [hunk 1]
```diff
@@ -269,7 +269,7 @@ bucket_put_simple:
             - '2 {garbage}'
         website_body:
             - null
-            - '<WebsiteConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><IndexDocument><Suffix>{suffix}</Suffix><IndexDocument>{error_doc}<WebsiteConfiguration/>'
+            - '<WebsiteConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><IndexDocument><Suffix>{suffix}</Suffix><IndexDocument>{error_doc}</WebsiteConfiguration>'
```
Comment (andrewgaul) [inline request_decision_graph.yml]: Close `IndexDocument` tag?

Comment (robbat2) [inline request_decision_graph.yml]: @andrewgaul fixed further in this request.

**Comment by andrewgaul** [comment]
Could you break out the unique test name into a separate PR?  Also squash the two website commits and ensure it has the signed off by line.

**Comment by robbat2** [comment]
Ok, will re-open

**Pull request closed without merging**